src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -190,39 +251,33 @@ export const WorkspaceInitial = () => {
   return (
     <EuiPage style={{ minHeight: '100vh' }}>
       <EuiPageBody>
-        <EuiFlexGroup direction="column" alignItems="center" style={{ paddingTop: '100px' }}>
-          <EuiFlexItem grow={false}>
-            <EuiImage size="l" alt="OpenSearch" src={logos.OpenSearch.url} />
-          </EuiFlexItem>
-          <EuiSpacer />
-          <EuiFlexItem grow={false} style={{ width: '1200px' }} className="eui-displayInline">
-            <EuiPageContent
-              style={{
-                backgroundImage: `url(${backGroundUrl})`,
-                backgroundSize: '490px 380px',
-                backgroundRepeat: 'no-repeat',
-                backgroundPosition: 'right 10px top 20px',
-              }}
-            >
-              {content}
-            </EuiPageContent>
-            <EuiSpacer size="m" />
-            <EuiSmallButtonEmpty
-              iconType="gear"
-              iconSide="left"
-              flush="left"
-              href={settingsAndSetupUrl}
-              data-test-subj="workspace-initial-button-settingsAndSetup"
-            >
-              <EuiText size="s">
-                {i18n.translate('workspace.initial.button.settingsAndSetup', {
-                  defaultMessage: 'Settings and setup',
-                })}
-              </EuiText>
-            </EuiSmallButtonEmpty>
+        <EuiIcon type="logoOpenSearch" size="xl" />
+        <EuiFlexGroup
+          direction="column"
+          alignItems="center"
+          justifyContent="flexStart"
+          style={{ paddingTop: '24px' }}
+        >
+          <EuiFlexItem grow={false} style={{ width: '1264px' }}>
+            {content}
           </EuiFlexItem>
         </EuiFlexGroup>
+
+        <div className="fixedLeftBottomIcon">
+          <div ref={mountSettingRef} />
+          <EuiSpacer size="s" />
+          <div ref={mountDevToolsRef} />
+          <EuiSpacer size="s" />
+          <div ref={mountUserAccountRef} />
+        </div>
       </EuiPageBody>
+      {isUseCaseFlyoutVisible && (
+        <WorkspaceUseCaseFlyout
```

Comment:
![image](https://github.com/user-attachments/assets/618d4764-4b08-49cf-877b-d9d8db711b03)

The features inside the flyout do not looks correct to me.

Reply:
Could you please provide more details？

Reply:
Different use cases should have different features, but from the screenshot above, it seems they all look the same. Is it expected?

Reply:
Yes, according the mockup in the create workspace page, They look the same.
![image](https://github.com/user-attachments/assets/827a9654-ff13-4e78-94c9-92f188fbb409)

Reply:
There is `configure` in the first use case but not in the second one. They should be different right?

Reply:
Yes, when i installed the maps plugin, we can see the differences between Observability and Security Analytics.
<img width="420" alt="image" src="https://github.com/user-attachments/assets/e97f9ba0-0bfe-4003-8f3b-a76dd719f42c">

Reply:
Oh, then we are good. Thanks for the clarification.